### PR TITLE
JS Sync tutorial

### PR DESCRIPTION
**Pull request by trodrigues**
First draft of the JS Sync tutorial. This is also somewhat dependent on this issue https://github.com/contentful/contentful.js/issues/65

### CODE REVIEW

**Comment by realityking** [comment]
Tutorial should also be linked here: https://github.com/contentful/slash-developers/blob/master/docs/concepts/sync.md

File: docs/javascript/tutorials/using-the-sync-api-with-js.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  console.log(response.deletedEntries)
+  console.log(response.deletedAssets)
+  // store the new token
+  window.localStorage.setItem('contentfulSyncToken', response.nextSyncToken)
+})
+~~~
+
+Every time you perform a sync you get a new token, which represents that point in time for your space, so don't forget to store it again.
+
+If you can then loop through the content you have previously stored and removed any content that is now marked as deleted, but that is left as an exercise to the reader.
```
Comment (realityking) [inline docs/javascript/tutorials/using-the-sync-api-with-js.md]: The beginning of the sentence seems weird `If you can then loop`, missing comma maybe?

**Comment by realityking** [comment]
Other than that :+1:

File: docs/javascript/tutorials/using-the-sync-api-with-js.md (hunk 1 above)
Comment (trodrigues) [inline docs/javascript/tutorials/using-the-sync-api-with-js.md]: Nah, the "If" should just not be there. I was probably writing something else and didn't rewrite it properly. Nice catch.

**Pull request merged**